layout/section.scss

```diff
@@ -0,0 +1,130 @@
+                            cursor: pointer;
+                        }
+                    }
+                }
+                .quantity {
+                    // margin-right: 60px;
+                    text-align: center;
+                    padding: 10px 0;
+                    input {
+                        border: none;
+                        text-align: center;
+                        width: 32px;
+                        font-size: 16px;
+                        color: #43484D;
+                        font-weight: 300;
+                      }
+                      button[class*= -btn] {
+                        // sẽ có nhiều button đều có class btn chung thế này, cần đặt css rõ chức năng hơn, cũng k nen set cứng width heigt
+                        padding: 10px 10px;
+                        background-color: #E1E8EE;
+                        border-radius: 6px;
+                        border: none;
+                        cursor: pointer;
+                        .fa-plus, .fa-minus {
+                            margin: 0;
+                        }
+                      }
+                }
+            }
+        }
+    }
+    .select-footer{
+        display: flex;
+        justify-content: space-around;
+        margin-top: 25px;
+        .select-footer-coupon{
+            .form-control{
+                padding: 10px 20px;
+                border: 1px solid #f5f7f6;
+            }
+            .btn-apply{
+                @extend .btn-apply
+            }
+        }
+        .float-right{
+            .btn-update, .btn-checkout{
+                padding: 10px 20px;
+                margin-right: 10px;
+                font-weight: bold;
+                color: #fff;
+                border: none;
+                border-radius: 5px;
+                &:hover{
+                    cursor: pointer;
+                    opacity: 0.8;
+                }
+            }
+            .btn-update{
+                background-color: #00bcd4;
+            }
+            .btn-checkout{
+                background-color: #393d50
+            }
+        }
+    }
+    .select-total{
+        display: flex;
+        justify-content: space-around;
+        margin-top: 30px;
+        .calculate{
+            h1{
+               @extend .getStyleHeader;
+            }
+            .calculate-div-btn{
+                margin-top: 10px;
+                .calculate-btn{
+                    @extend .btn-apply;
+                    margin: 0;
+                }
+            }
+        }
+    }
+    .selector-total-cart{
+        padding: 0 90px;
+        h1 {
+            @extend .getStyleHeader;
+        }
+        table{
+            tr{
+                .title{
+                    padding: 10px 20px;
+                    background-color: #999;
+                    color: #333;
+                }
+                .price{
+                    padding-left: 10px;
+                }
+            }
+        }
+    }
+}
```

Comment:
sửa tất cả các cmt cũ, áp dụng cho toàn bộ các file nha, xong đẩy pull nữa a xem